src/harrastuspassi/harrastuspassi/models.py

```diff
@@ -204,11 +205,34 @@ def __str__(self):
 
 
 class Promotion(TimestampedModel):
-    hobby = models.ForeignKey(Hobby, on_delete=models.CASCADE)
     name = models.CharField(max_length=1024)
     description = models.TextField()
     start_date = models.DateField(blank=False, null=False, verbose_name=_('Start date'))
     start_time = models.TimeField(blank=False, null=False, verbose_name=_('Start time'))
     end_date = models.DateField(blank=False, null=False, verbose_name=_('End date'))
     end_time = models.TimeField(blank=False, null=False, verbose_name=_('End time'))
     cover_image = models.ImageField(upload_to='promo_images', null=True, blank=True)
+    municipality = models.ForeignKey(Municipality, null=True, blank=True, on_delete=models.CASCADE)
+    organizer = models.ForeignKey(Organizer, on_delete=models.CASCADE)
+    available_count = models.PositiveIntegerField()
+    used_count = models.PositiveIntegerField(default=0)
+    location = models.ForeignKey(Location, on_delete=models.CASCADE)
+
+    def __str__(self):
+        return self.name
+
+
+class Benefit(TimestampedModel):
+    promotion = models.ForeignKey(Promotion, on_delete=models.CASCADE)
```

Comment:
There could be a short docstring on what Benefit and Promotion are and how they're related and why. Doesn't have to go into implementation details, since those will change often.